src/chunkserver/datastore/file_pool.h

```diff
@@ -202,7 +207,9 @@ class CURVE_CACHELINE_ALIGNMENT FilePool {
     /**
      * @brief: Return the suffix of clean chunk
      */
-    static std::string GetCleanChunkSuffix();
+    inline static std::string GetCleanChunkSuffix() {
```

Comment:
keyword `inline` is redundant because member functions defined in header files are implicit inline.

Reply:
@wu-hanqing done.